### DISCUSSION TITLE
PYG to ONNX

### DISCUSSION THREAD

**Discussion by CY-SCUT** [pyg-team/pytorch_geometric#8154] [Q&A]
Can you please tell me what cannot be used after converting PYG to Onnx now? Or are there any existing issues total?

**Comment by rusty1s** [comment]
I don't have a full coverage report, but we test our standard GNN layers with it (e.g, `GCN`, `GraphSAGE`, `GAT`, ...) and they seem to be converting just fine to ONNX.
